src/main/io/vtx_tramp.h

```diff
@@ -36,8 +36,10 @@
 #define VTX_TRAMP_MIN_FREQUENCY_MHZ 5000             //min freq in MHz
 #define VTX_TRAMP_MAX_FREQUENCY_MHZ 5999             //max freq in MHz
 
+#if 0
```

Comment:
Please comment on when this will be re-enabled or remove.

Reply:
Removed ;)